paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityShouldBurn.java

```diff
@@ -16,12 +17,13 @@ public class EntityShouldBurn extends EntityProperty<ElementTag> {
     // @input ElementTag(Boolean)
     // @plugin Paper
     // @description
-    // If the entity is a Zombie, Skeleton, or Phantom, controls whether it should burn in daylight.
+    // If the entity is a Zombie, Skeleton, Stray, or Phantom, controls whether it should burn in daylight.
     // -->
 
     public static boolean describes(EntityTag entity) {
         return entity.getBukkitEntity() instanceof Zombie
                 || entity.getBukkitEntity() instanceof Skeleton
+                || entity.getBukkitEntity() instanceof Stray

```

Comment:
Should this just be `AbstractSkeleton` (covering both Skeleton and Stray), which is the interface that the method is on anyway?